dbtype/bigint.go

```diff
@@ -175,3 +178,39 @@ func (b *BigInt) Scan(src interface{}) error {
 
 	return nil
 }
+
+// func (src *Point) AssignTo(dst interface{}) error {
+// 	return fmt.Errorf("cannot assign %v to %T", src, dst)
+// }
+
+func (b BigInt) DecodeText(ci *pgtype.ConnInfo, src []byte) error {
```

Comment:
You can ignore this method, it was just me doing some hacking code. I was thinking that if pgx cannot support a null value of a normal database/sql scanner+valuer, then we can try the add binary encoding support here which might solve our issue in the end.

Alternatively, maybe there is another way to make pgx or scany work, and if not, finally we could adapt pgkit instead of using the pgx-direct driver, we update all of pgkit to use the database/sql interface which pgx also supports. I'm not sure exactly what we'd compromise by not using pgx-driver directly anymore, but it would be nice to know that too if we decide that..